_data/navigation.yml

```diff
@@ -82,6 +82,8 @@ wiki:
         url: /wiki/sensing/opencv-stereo/
       - title: Camera-IMU Calibration using kalibr
         url: /wiki/sensing/camera-imu-calibration/
+      - title: RTK GPS
+	url: /wiki/sensing/gps/
```

Comment:
Double check the white space here. Should be spaces and the 'u' of url should line up with the 't' of title.